plugins/sample/plugin_sample.c

```diff
@@ -57,22 +57,22 @@ PLUGIN_EXPORT bool sir_plugin_query(sir_plugininfo* info) {
     info->desc      = desc;
     info->caps      = caps;
 
-    printf("\t" DGRAY("plugin_sample ('%s')") "\n", __func__);
+    (void)printf("\t" DGRAY("plugin_sample ('%s')") "\n", __func__);
     return true;
 }
 
 PLUGIN_EXPORT bool sir_plugin_init(void) {
-    printf("\t" DGRAY("plugin_sample ('%s')") "\n", __func__);
+    (void)printf("\t" DGRAY("plugin_sample ('%s')") "\n", __func__);
     return true;
 }
 
 PLUGIN_EXPORT bool sir_plugin_write(sir_level level, const char* message) {
-    printf("\t" DGRAY("plugin_sample (%s): level: %04"PRIx32", message: %s"),
-        __func__, level, message);
+    (void)printf("\t" DGRAY("plugin_sample (%s): level: %04"PRIx32", message: %s"),
+                 __func__, level, message);
     return true;
 }
 
 PLUGIN_EXPORT bool sir_plugin_cleanup(void) { //-V524
-    printf("\t" DGRAY("plugin_sample ('%s')") "\n", __func__);
+    (void)printf("\t" DGRAY("plugin_sample ('%s')") "\n", __func__);
```

Comment:
which compiler warning flag caused this to be marked as a warning?

Reply:
That's Oracle Lint